src/components/TmLiStakeTransaction/TmLiStakeTransaction.vue

```diff
@@ -8,6 +8,20 @@ tm-li-transaction(:color="color" :time="transaction.time" :block="transaction.he
     div(slot="details")
       | To&nbsp;
       router-link(:to="this.validatorURL + '/' + tx.validator_addr") {{moniker(tx.validator_addr)}}
+  template(v-if="redelegation")
+    div(slot="caption")
+      | Redelegated&nbsp;
+      template(v-if="transaction.redelegation")
+        b {{transaction.redelegation.amount}}
+        span &nbsp;{{transaction.redelegation.denom.toUpperCase()}}S
+      template(v-else)
+        b {{tx.shares}}
+        span &nbsp;Shares
+    div(slot="details")
+      | From&nbsp;
+      router-link(:to="this.validatorURL + '/' + tx.validator_src_addr") {{moniker(tx.validator_src_addr)}}
+      |  to&nbsp;
+      router-link(:to="this.validatorURL + '/' + tx.validator_src_addr") {{moniker(tx.validator_dst_addr)}}
```

Comment:
validator_src_addr needs to be validator_dst_addr here

Reply:
good catch, thanks !